torch/cuda/jiterator.py

```diff
@@ -0,0 +1,88 @@
+import torch
+from torch import Tensor
+from typing import Callable
+
```

Comment:
should fail if cuda does not exist

Reply:
following cuda module's lazy initialization pattern. 
checked in __init__ function.